dashboard/test/controllers/api/v1/pd/workshop_summary_report_controller_test.rb

```diff
@@ -192,6 +192,21 @@ class Api::V1::Pd::WorkshopSummaryReportControllerTest < ::ActionController::Tes
     assert_equal EXPECTED_COMMON_FIELDS.count + EXPECTED_PAYMENT_FIELDS.count, response.first.count
   end
 
+  test 'includes unpaid workshops' do
+    unpaid_workshop = create :pd_ended_workshop, organizer: @organizer, course: Pd::Workshop::COURSE_CSD
+    create :pd_workshop_participant, workshop: @workshop, enrolled: true, in_section: true, attended: true
+
+    sign_in @admin
+    get :index
+    assert_response :success
+    response = JSON.parse(@response.body)
+    assert_equal 3, response.count
+    unpaid_report = response.find{|row| row['workshop_id'] == unpaid_workshop.id}
```

Comment:
Hound! At least they "notified" us this time... https://twitter.com/houndci/status/811271775904333825